rma_sale_mrp/models/account_move.py

```diff
@@ -16,7 +16,8 @@ def _check_rma_invoice_lines_qty(self):
         if lines:
```

Comment:
Maybe this could be redesigned:

- If there are `lines` from the super we return them directly (thus the error will be raised)
- Otherwise we filter again over `self.invoice_line_ids` with the kits conditions.

Reply:
Yes a redesign would be good. Done :-)

Reply:
Sorry no its not possible to return the lines because they maybe contain kit lines

Reply:
@chienandalu could you remove the stale label and merge this one?